packages/lab/stories/tokenized-input-next/tokenized-input-next.stories.tsx

```diff
@@ -0,0 +1,336 @@
+import {
+  Button,
+  FormField,
+  FormFieldHelperText as FormHelperText,
+  FormFieldLabel,
+  FormFieldLabel as FormLabel,
+  Input,
+} from "@salt-ds/core";
+import { TokenizedInputNext } from "@salt-ds/lab";
+import { Meta, StoryFn } from "@storybook/react";
+import {
+  ChangeEventHandler,
+  SyntheticEvent,
+  useEffect,
+  useLayoutEffect,
+  useRef,
+  useState,
+} from "react";
+import { statesData } from "../assets/exampleData";
+
+export default {
+  title: "Lab/Tokenized Input Next",
+  component: TokenizedInputNext,
+} as Meta<typeof TokenizedInputNext>;
+
+const NUMBER_REGEX = /^(|[1-9][0-9]*)$/;
+
+const getRandomNumber = () => Math.floor(Math.random() * 99 + 1);
+
+const TokenizedInputNextTemplate: StoryFn<typeof TokenizedInputNext> = (
+  props
+) => {
+  const handleChange = (
+    event: SyntheticEvent,
+    selectedItems: unknown[] | undefined
+  ) => {
+    console.log("selection changed", selectedItems);
+  };
+
+  return (
+    <TokenizedInputNext
+      style={{ width: "266px" }}
+      onChange={handleChange}
+      {...props}
+    />
+  );
+};
+
+export const Default = TokenizedInputNextTemplate.bind({});
+Default.args = {};
+
+export const WithCollapsedButton = TokenizedInputNextTemplate.bind({});
```

Comment:
This example looks different before and after expanding (can't add sc on vpn) but the first time I visit: https://fc6c9269.saltdesignsystem-storybook.pages.dev/?path=/story/lab-tokenized-input-next--with-collapsed-button Arkansas is in the row and it's not aligned, but then I expand and collapse and it's aligned

Reply:
After some debugging it looks like `dimension = node.getBoundingClientRect();` in `getWidth` is initially returning a number that is smaller than a width of the pill, but after expanding and collapsing it is correct.

Not sure if you will quickly know what is wrong, otherwise in Tabs we are using https://react.fluentui.dev/?path=/docs/components-overflow--default to handle this kind of visibility problem

Reply:
thanks! I'll have a look